<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerOpenIGTLinkRemoteCommandWidget</class>
 <widget class="qSlicerWidget" name="qSlicerOpenIGTLinkRemoteCommandWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>OpenIGTLink connector</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="qMRMLNodeComboBox" name="ConnectorComboBox">
           <property name="nodeTypes">
            <stringlist>
             <string>vtkMRMLIGTLConnectorNode</string>
            </stringlist>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="commandGroupBox">
        <property name="title">
         <string>Command</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPlainTextEdit" name="CommandTextEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="SendCommandButton">
           <property name="text">
            <string>Send command</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="responseGroupBox">
        <property name="title">
         <string>Response received</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QPlainTextEdit" name="ResponseTextEdit"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>qSlicerOpenIGTLinkRemoteCommandWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>ConnectorComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>396</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
